### PR TITLE
add kyrgyz language 

### PR DESCRIPTION
**Pull request by g1039**

### CODE REVIEW

**Comment by g1039** [comment]
screenshot

**Comment by g1039** [comment]
@miltontony @saeedmb8 Please review

File: locale/ky/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -0,0 +1,513 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+#, fuzzy
```
Comment (smarzban) [inline locale/ky/LC_MESSAGES/django.po]: Please remove `fuzzy`

File: locale/ky/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -0,0 +1,513 @@
...
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2016-09-13 23:13+0200\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"Language: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=INTEGER; plural=EXPRESSION;\n"
```
Comment (smarzban) [inline locale/ky/LC_MESSAGES/django.po]: Please remove this line as well

**Comment by g1039** [comment]
@saeedmb8 Ready for re-review

**Comment by smarzban** [comment]
👍

**Pull request merged**